lib/src/main/java/com/tubitv/media/activities/TubiPlayerActivity.java

```diff
@@ -27,7 +28,7 @@
  * Along with some abstract methods to be implemented by subclass for extra functions.
  * You can use this class as it is and implement the abstract methods to be a standalone player to player video with customized UI controls and different forms of adaptive streaming.
  */
-public abstract class TubiPlayerActivity extends LifeCycleActivity
```

Comment:
Why does this no longer need to be a LifeCycleActivity?

Reply:
This is for RX binding. LifeCycleActivity has different lifecycle handling so I have to get rid of it and match with what we use now